assignments/prototypes.js

```diff
@@ -41,8 +41,55 @@
 
 // Test you work by un-commenting these 3 objects and the list of console logs below:
 
-/*
-  const mage = new Humanoid({
+function GameObject(gameObjectItem) {
+  this.createdAt = gameObjectItem.createdAt;
+  this.dimensions = gameObjectItem.dimensions;
+};
+
+GameObject.prototype.destroy = function () {
+  return `Object was removed from the game.`
+};
+
+function CharacterStats(statsItem) {
+  GameObject.call(this, statsItem);
+  this.healthPoints = statsItem.healthPoints;
+  this.name = statsItem.name;
+};
+
```

Comment:
Pop this up here instead...

```suggestion
CharacterStats.prototype = Object.create(GameObject.prototype);
```